client/app/pages/dashboards/dashboard.js

```diff
@@ -448,14 +448,14 @@ function DashboardCtrl(
 
   this.openShareForm = () => {
     // check if any of the wigets have query parameters
```

Comment:
This comment should be removed or updated.

Reply:
https://twitter.com/nzkoz/status/538892801941848064?s=20